src/main/kotlin/se/uulm/snowballr/backend/service/ProjectService.kt

```diff
@@ -96,6 +96,11 @@ interface IProjectService {
      * Service implementation of [SnowballRService.getPapersToReviewForProject]
      */
     suspend fun getPapersToReviewForProject(request: Base.Id): GrpcProject.Paper.List
+
+    /**
+     * Service implementation of [SnowballRService.addPaperToProject]
```

Comment:
```suggestion
     * Service implementation of [SnowballRService.addPaperToProject].
```

In the comment for the call above too.